### PR TITLE
Use localized service descriptions in logbook

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Screen Shot 2022-06-03 at 20 12 40

Screen Shot 2022-06-03 at 17 07 20
Screen Shot 2022-06-03 at 20 08 57

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by spacegaier** [comment]
Two remarks/questions:
1. The service names will still always be in EN, correct? No translation for those?
2. I am wondering about the formatting, esp. whether the domain and the service name should be visually more connected. In EN you kind of see that the domain and service name belong together since those are in capital letters, but e.g. in German the word service (= "Dienst") is also in capitals.

Just toying with some ideas...
> Wohnen Fenster wurde geschlossen ausgelöst durch einen Dienst Abdeckung: Close

> Wohnen Fenster wurde geschlossen ausgelöst durch einen Dienst "Abdeckung: Close"

> Wohnen Fenster wurde geschlossen ausgelöst durch einen Dienst (Abdeckung: Close)

**Comment by bdraco** [comment]
> Two remarks/questions:
> 
> 1. The service names will still always be in EN, correct? No translation for those?

Not yet AFAICT but now closer to possible 
> 
> 2. I am wondering about the formatting, esp. whether the domain and the service name should be visually more connected. In EN you kind of see that the domain and service name belong together since those are in capital letters, but e.g. in German the word service (= "Dienst") is also in capitals.
> 
> 
Feel free to push a change to this branch if you've got something that looks cleaner 
> 
> Just toying with some ideas...
> 
> > Wohnen Fenster wurde geschlossen ausgelöst durch einen Dienst Abdeckung: Close
> 
> 
> 
> > Wohnen Fenster wurde geschlossen ausgelöst durch einen Dienst "Abdeckung: Close"
> 
> 
> 
> > Wohnen Fenster wurde geschlossen ausgelöst durch einen Dienst (Abdeckung: Close)

**Comment by balloob** [comment]
Seeing all these examples, it makes me wonder if we need to even add if it was changed by a service ? If we can attribute it to a user or automation, that would be helpful. 

I guess we might want to know is if it is happening via HA versus a state change that happened because we fetched new data. 

CC @matthiasdebaat

**Comment by bdraco** [comment]
Note to future self: The service call is what provides the user id from the context.  If we dropped that we would need to make sure we still set the user on the row

File: src/data/service.ts [hunk 1]
```diff
@@ -12,3 +13,18 @@ export const callExecuteScript = (
...
+export const serviceName = (
+  hass: HomeAssistant,
+  domain?: string,
+  service?: string
+) =>
+  domain &&
+  domain in hass.services &&
+  service &&
+  service in hass.services[domain] &&
+  hass.services[domain][service].name
```
Comment (bramkragten) [inline src/data/service.ts:26]:
```suggestion
  hass.services[domain]?.[service]?.name
```

**Review by bramkragten — DISMISSED** [review]

**Pull request closed without merging**